packages/workbox-routing/lib/Route.mjs

```diff
@@ -0,0 +1,170 @@
+/*
+ Copyright 2017 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import core from 'workbox-core';
+
+import {defaultMethod, validMethods} from './constants.mjs';
+
+/**
+ * This is the definition of the `match` callback passed into the
+ * `Route` constructor.
+ *
+ * This callback is used to determine if a new `fetch` event can be served
+ * by this `Route`. Returning a truthy value indicates that this `Route` can
+ * handle this `fetch` event. Return `null` if this shouldn't match against
```

Comment:
This may be an implementation detail or a comment detail but presumably a "non-truthy" value will indicate the `Route` can't handle the fetch event.

If so would 'Return `false`' be better than 'Return `null`' just for simplicity?

Reply:
I think the idea is that someone might have a `match` function that wants to forward the parameter `false` to the handler. Pretty edge-casey, but I think most of the forwarding parameters stuff is not particularly used.

I'd rather just keep things as-is and match the v2 behavior, though.

Reply:
But returning false isn't a truthy value, so which of the statements is correct, return a truthy value or return anything other than null?

Reply:
Oh, gotcha.

It should be "return anything other than `null`". I'll adjust the JSDoc.